.github/workflows/configure.sh

```diff
@@ -19,7 +19,7 @@ export color=no
 rm -rf ${top}/install
 if [ "${maker}" = "make" ]; then
     make distclean
-    make config CXXFLAGS="-Werror" prefix=${top}/install \
```

Comment:
We need the `-Werror` flag. If there are warnings that are being generated, we need to know about and fix those warnings, not remove the flag.

Reply:
Yes, I wanted to discuss this.  There is a problem with icpx which causes it to automatically apply some flags at compile time which are only applicable during linking (e.g. rpath).  When compiling with -Werror, the extra flags cause it to exit with an error.  The extra flags are applied based on the icpx.cnf which is part of the intel-oneapi-compilers package installation.  I'm not sure what to do about this other than to remove -Werror. Suggestion?

Reply:
Does adding `-Wno-unused-command-line-argument` help? Seems g++, clang++, icpx all accept that flag.
```
sh leconte test> icpx -Wall -c -o hello_cc.o hello_cc.cc 
icpx: warning: -Wl,-rpath,/nfs/apps/spacks/2023-01-01/opt/spack/linux-centos7-x86_64/gcc-9.5.0/intel-oneapi-mkl-2023.0.0-ahun4a3x3lfkmhp3hnztwufywmkyrqza/mkl/2023.0.0/lib/intel64: 'linker' input unused [-Wunused-command-line-argument]
sh leconte test> icpx    -Wall -Wno-unused-command-line-argument -c -o hello_cc.o hello_cc.cc 
sh leconte test> g++     -Wall -Wno-unused-command-line-argument -c -o hello_cc.o hello_cc.cc 
sh leconte test> clang++ -Wall -Wno-unused-command-line-argument -c -o hello_cc.o hello_cc.cc 
sh leconte test> 
```

Reply:
Yes I think that would resolve the issue.  I'll try it here.